lib/src/pages/splash.dart

```diff
@@ -15,11 +15,14 @@ class SplashScreen extends StatelessWidget {
     ]);
 
     Future.delayed(const Duration(seconds: 2), () async {
-      bool isLoggedIn = await UserController().isUserLoggedIn;
+      bool isLoggedIn = await UserValidation().checkUserLogin();
       Navigator.of(context).push(
         MaterialPageRoute(
-            builder: (context) => isLoggedIn ? const HomePage(title: 'Help a Paw') : const LoginPage(),
-            settings: RouteSettings(name: isLoggedIn ? global.homeRoute : global.loginRoute)),
+            builder: (context) => isLoggedIn
```

Comment:
We should actually always go to the home screen, even if user is not logged in

Reply:
Fixed